docs/howto/features/rocker.md

```diff
@@ -128,4 +128,13 @@ jupyterhub:
               slug: scipy
               kubespawner_override:
                 image: jupyter/scipy-notebook:2023-06-26
-```
\ No newline at end of file
+```
+
+## User installed packages
+
+In R, users can install packages with `install.packages` or similar R commands. Rocker
+is configured to automatically install packages from the [Posit Package Manager](https://packagemanager.posit.co/client/#/),
+which provides fast binary package installs.
+
+They are installed under `/usr/local`, and hence *cleared on server stop*. So installed
+packages will *not* persist across user sessions. 
```

Comment:
Maybe it's worth adding a line about how to make them persistent, i.e. adding them to the image.